news/2022-12-15-project-loved-december-2022.md

```diff
@@ -0,0 +1,372 @@
+
+This chart of *Frontier Explorer* has a variety of delay and chordstream patterns. There are many rates for those of various skill levels, ranging from those getting into reading very dense chordstreams at a slower BPM, to the greatest ETs like Blocko himself!
+
+While this was originally uploaded to celebrate New Year 2021, there is also a custom 1.05x rate difficulty featured in the Grand Finals pool of [osu!mania 7K World Cup 2022](/wiki/Tournaments/MWC/2022_7K) featuring only chordstreams. The set as a whole garnered quite a bit of popularity among the people who are capable of playing this.
+
+I don't recommend that you pause for this, as stamina will factor exceptionally heavily in your attempts at this chart. Good luck, and don't forget to vote for *Frontier Explorer*!
+
+[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/1264121.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[YOASOBI - Ano Yume o Nazotte](https://osu.ppy.sh/beatmapsets/1264121#mania) by [Remuring](https://osu.ppy.sh/users/6522146) and [Pengdoll](https://osu.ppy.sh/users/6392061)\
+180 BPM, 3:59 | 7.69★\
+*description written by [-NoName-](https://osu.ppy.sh/users/10400730)*
+
+Our very own [Remuring](https://osu.ppy.sh/users/6522146), who we recently welcomed to the team of osu!mania captains last month, has a map featured in this month's Loved voting: *Ano Yume o Nazotte*!
+
+Created in collaboration with [Pengdoll](https://osu.ppy.sh/users/6392061), this has mapping styles you would typically see from both of them. We captains aim to kill two birds with one stone with this nomination, and we feel that this delivers some of the best charting from both creators here. Typical to LN charts like these, you'll see a lot of notes in between columns occupied by LNs, which is great for increasing your finger strength overall.
+
+Aside from the above, we know that many people in the community are fans of YOASOBI's music, so we know those fans will be happy to see another YOASOBI chart in the Loved voting. If you want to see a good LN collaboration practice chart in Loved, or you just want to jam to YOASOBI, remember to give *Ano Yume o Nazotte* your support!
+
+---
+
+From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!
+
+More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). Stop by the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy) if you have any questions, concerns, or are interested in helping out.
```

Comment:
```Suggestion
More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). If there are any questions, comments, concerns, or you are interested in helping out, please stop by the `#osu-loved` channel in the [osu!dev Discord server](https://discord.gg/ppy).
```

Reply:
prefer current wording, we already used it many times too